### DISCUSSION TITLE
Solid-Start Project can't be deployed somehow.

### DISCUSSION THREAD

**Discussion by atzzCokeK** [vercel/vercel#8944] [Help] [answered]
Hi, I need your help...
I don't know why, but my project suddenly can't be deployed. If I run deploying, `solid-start rendering index.html...` never ends and in 45 mins failed.
I can build locally and deploy with Vercel CLI, but it can't be promoted to Production. Does anyone know how to deal with it? 
Do I better ask the SolidJS community as well?
![image](https://user-images.githubusercontent.com/25091548/202615369-886a5225-d239-4fa7-ad34-41021e426773.png)

I have already checked deploy command settings. It does't seem bad to me.
![image](https://user-images.githubusercontent.com/25091548/202616105-4b3e5cf1-409e-4ba4-9248-e685751570c0.png)

I am using `"solid-start-vercel": "^0.2.5"`

**Comment by amyegan** [comment, marked as answer]
Hi @atzzCokeK! If I understand you correctly, you're trying to promote a deployment from the project dashboard and seeing this error
```
Error: Only Deployments created via Git can be promoted to production. 
```

[Non-git deployments cannot be promoted](https://vercel.link/promote-non-git-deployment) to production from the dashboard. When using the CLI, you can create a production deployment from the same code used for your preview by deploying with [`vercel --prod`](https://vercel.com/docs/cli/deploy#prod). I know the team is working on some changes to let you promote CLI preview deployments, but I don't have an ETA on that yet.
